inc/WebhookHandler/GitHub.php

```diff
@@ -117,24 +118,25 @@ public function callback(): \WP_Error|\WP_REST_Response {
 		 * @var array{repository: array{ default_branch?: string, html_url: string, full_name: string, ssh_url: string, clone_url: string, private: bool }, ref: string } $params
 		 */
 
-		if ( ! isset( $params['repository']['default_branch'] ) ) {
-			return new \WP_Error( '400', 'Request incomplete', [ 'status' => 400 ] );
-		}
```

Comment:
This check is still needed because the `default_branch` array offset might not exist according to the docblock above.